apollo-android-support/build.gradle.kts

```diff
@@ -11,8 +11,8 @@ extensions.findByType(BaseExtension::class.java)!!.apply {
   compileSdkVersion(groovy.util.Eval.x(project, "x.androidConfig.compileSdkVersion").toString().toInt())
 
   compileOptions {
-    sourceCompatibility = JavaVersion.VERSION_1_7
-    targetCompatibility = JavaVersion.VERSION_1_7
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
```

Comment:
I think this is ok but I guess it's worth double checking and give a warning in release notes if needed. I started a thread over there: https://github.com/apollographql/apollo-android/issues/2057

Reply:
Google nowadays recommends libraries to target Java 8 to move the platform forward. Very few of the AndroidX libs already do that. OkHttp too. 

Technically we don't have to do it for all modules. But for consistency that would be good. I will check if this can be done in root.